crates/key-value-azure/src/store.rs

```diff
@@ -216,24 +228,67 @@ impl Store for AzureCosmosStore {
         Ok(())
     }
 
+    // WARNING: this function only works on the primary region because the
```

Comment:
Nit: I think it's preferable to make this a doc comment (rather than a plain comment) so it's more easily visible outside of the source code.